src/main/resources/application.properties

```
@@ -1,6 +1,7 @@
-spring.datasource.url=jdbc:postgresql://localhost:5432/inaam
-spring.datasource.username=postgres
-spring.datasource.password=postgres
+spring.datasource.url=jdbc:postgresql://${SPRING_DATASOURCE_HOST}:${SPRING_DATASOURCE_PORT}/${SPRING_DATASOURCE_DATABASE}
```

Allow usage of other DB drivers.

```suggestion
spring.datasource.url=jdbc:${SPRING_DATASOURCE_DRIVER}://${SPRING_DATASOURCE_HOST}:${SPRING_DATASOURCE_PORT}/${SPRING_DATASOURCE_DATABASE}
```

---

Added usage of other DB drivers